deployments/puppet/README.md

```diff
@@ -63,6 +63,12 @@ This class accepts the following parameters:
+| `auto_instrumentation_version` | **Linux only**: Version of the `splunk-otel-auto-instrumentation` package to install, e.g. `0.50.0`. The minimum supported version is `0.48.0`. **Note:** The Java application on the node needs to be restarted separately in order for any change to take effect. | `latest` |
+| `auto_instrumentation_ld_so_preload` | **Linux only**: By default, the `/etc/ld.so.preload` file on the node will be configured for the `/usr/lib/splunk-instrumentation/libsplunk.so` [shared object library](https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation#operation) provided by the `splunk-otel-auto-instrumentation` package and is required for auto instrumentation. Configure this variable to include additional library paths, e.g. `/path/to/my.library.so`. **Note:** The Java application on the node needs to be restarted separately in order for any change to take effect. | None |
+| `auto_instrumentation_java_agent_jar` | **Linux only**: Path to the [Splunk OpenTelemetry Java agent](https://github.com/signalfx/splunk-otel-java). The default path is provided by the `splunk-otel-auto-instrumentation` package. If the path is changed from the default value, the path should be an existing file on the node. The specified path will be added to the `/usr/lib/splunk-instrumentation/instrumentation.conf` config file on the node. **Note:** The Java application on the node needs to be restarted separately in order for any change to take effect. | `/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar` |
+| `auto_instrumentation_resource_attributes` | **Linux only**: Configure the OpenTelemetry instrumentation [resource attributes](https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation#configuration-file), e.g. `deployment.environment=prod`. The specified resource attribute(s) will be added to the `/usr/lib/splunk-instrumentation/instrumentation.conf` config file on the node. **Note:** The Java application on the node needs to be restarted separately in order for any change to take effect. | None |
+| `auto_instrumentation_service_name` | **Linux only**: Explicitly set the [service name](https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation#configuration-file), for the instrumented Java application, e.g. `my.service`. By default, the service name is automatically derived from the arguments of the Java executable on the node. The specified service name will be added to the `/usr/lib/splunk-instrumentation/instrumentation.conf` config file on the node, overriding any generated service name. **Note:** The Java application on the node needs to be restarted separately in order for any change to take effect. | None |
```

Comment:
Perhaps something like this?

```suggestion
| `auto_instrumentation_service_name` | **Linux only**: Explicitly set the [service name](https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation#configuration-file), for the instrumented Java application, e.g. `my.service`. By default, the service name is automatically derived from the arguments of the Java executable on the node. If, however, the service name is explicitly specified, it will be added to the `/usr/lib/splunk-instrumentation/instrumentation.conf` config file on the node, overriding any generated service name. **Note:** The Java application on the node needs to be restarted separately in order for any change to take effect. | None |
```